### ISSUE TITLE
SHOWSTOPPER: On Windows 10, registry key under NativeMessagingHosts not written

### ISSUE THREAD

**Issue by aecreations** [aecreations/sync-clippings#2]
Sync Clippings 1.0b1

Issue with installer on Windows 10. The registry key HKLM\Software\Mozilla\NativeMessagingHosts\syncClippings is not being written. This issue is not occurring on Windows 7.

**Comment by aecreations** [comment]
http://nsis.sourceforge.net/Docs/Chapter4.html#setregview

**Comment by aecreations** [comment]
Suggested fix:
```
!include LogicLib.nsh
!include x64.nsh
# .
# .
# .
${If} ${RunningX64}
 # TEMPORARY
 SetRegView 32 # Force 32-bit mode
 # END TEMPORARY

 # Enable this once a 64-bit version of the native app is generated.
  #SetRegView 64
${Else}
  SetRegView 32
${EndIf}

**Comment by aecreations** [comment]
Commit - a6563c7f5dcce924b37f14df5ba919ee5fcaf288 - Changed script to build a 64-bit installer/uninstaller

A separate script is needed to build a 32-bit installer. Sigh.

**Comment by aecreations** [comment]
Fixed in 1.0 beta 2
